CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,31 @@
+# NoOps Code of Conduct v1.0 #
+
+[日本語版](#日本語版) below
+
+# Introduction
+NoOps Community is dedicated to providing a harassment-free experience for participants at all of our events. NoOps Community events are working conferences intended for professional networking and collaboration. They exist to encourage the open exchange of ideas and expression and require an environment that recognizes the inherent worth of every person and group. While at NoOps Community events or related ancillary or social events, any participants, including speakers, attendees, volunteers, sponsors, exhibitors, booth staff and anyone else, should not engage in harassment in any form.
+
+# Expected Behavior
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+# Unacceptable Behavior
+- The use of sexualized language or imagery and unwelcome sexual attention or advances
+- Trolling, insulting/derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others’ private information, such as a physical or electronic address, without explicit permission
+- Other conduct which could reasonably be considered inappropriate in a professional setting
+
+# Scope
+This Code of Conduct applies within project spaces and in public spaces when an individual is representing the project or its community, speakers, attendees, volunteers, sponsors, exhibitors, booth staff and anyone else. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+# Consequences
+Understand that speech and actions have consequences, and unacceptable behavior will not be tolerated.
+If you are the subject of, or witness to any violations of this Code of Conduct, please contact us by submitting a form here, or email xxx@xxxx.
```

Comment:
- Do we have a link for a contact form in `here` part? 
- It's better to add a specific email address instead of `xxx@xxxx`

Reply:
updated to community email